src/components/commons/SkipToContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const SkipToContent = () => {
+  const { t } = useTranslation();
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
+
+  const linkStyle: any = {
+    transition: "opacity 0.2s ease-out",
+    position: "relative",
+    opacity: isFocused ? "1" : "0",
+    zIndex: isFocused ? "1" : "-1",
+  };
+
+  return (
+    <div style={{ position: "relative" }}>
+      <Button
+        href="#main_content"
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+        style={linkStyle}
+      >
```

Comment:
Also add `tabIndex={-1}` in order to show the user where the focus has moved to after SkipToContent click.

Reply:
Not possible
As per documentation this would disable tab navigation twards the skip-to-content button too, wich is against the scope of the task
![image](https://github.com/user-attachments/assets/577fa620-6eb3-423e-bc60-8bcd49df131e)

Reply:
It is correct that main content cannot be reached via tab navigation. This occurs only when you click the skipToContent button.

Reply:
done here e4eb02844ee93129becf1d979bc9a4e1dd6bfdee